iroh-resolver/src/unixfs_builder.rs

```diff
@@ -518,7 +633,10 @@ async fn make_dir_from_path<P: Into<PathBuf>>(path: P, recursive: bool) -> Resul
     let mut directory_reader = tokio::fs::read_dir(path.clone()).await?;
     while let Some(entry) = directory_reader.next_entry().await? {
         let path = entry.path();
-        if path.is_file() {
+        if path.is_symlink() {
+            let s = SymlinkBuilder::new(path).build().await?;
```

Comment:
missing target?

Reply:
Ah, so In `FileBuilder`, if you only give it a path, it will derive the `name` & a `reader` for you from that `path`. I matched that pattern here, if you only provide a `path` it will derive the `target` for you. Or you can add it explicitly using the `target` method.

Reply:
ah I see, makes sense